### PR TITLE
Some Initial Predefined Metrics

### LINKED ISSUE #309: Add loads of predefined metrics in symbolic module
🐞 **What bugs you?**
  - Have predefined metrics in *symbolic* module

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by ritzvik**
Partially Fixes #309

### CODE REVIEW

File: src/einsteinpy/symbolic/predef/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .de_sitter import AntiDeSitter, AntiDeSitterStatic, DeSitter
```
Comment (shreyasbapat) [inline src/einsteinpy/symbolic/predef/__init__.py]: Can we have something else than `predef` ? Any other ideas on a better word?  :confused:

Comment (ritzvik) [reply src/einsteinpy/symbolic/predef/__init__.py]: maybe `predefined` ? 
Anyways, `predef` seemed cute!

Comment (shreyasbapat) [reply src/einsteinpy/symbolic/predef/__init__.py]: LOL

Comment (shreyasbapat) [reply src/einsteinpy/symbolic/predef/__init__.py]: Any synonym? That sounds cool?

Comment (ritzvik) [reply src/einsteinpy/symbolic/predef/__init__.py]: no suitable synonym available. Search google!
going with `predefined`

Comment (ritzvik) [reply src/einsteinpy/symbolic/predef/__init__.py]: Resolved with commit 73830ca

**Pull request merged**